x-pack/plugins/actions/server/builtin_action_types/servicenow/utils.ts

```diff
@@ -134,7 +134,11 @@ export const getAxiosInstance = ({
           tokenUrl: `${snServiceUrl}/oauth_token.do`,
           connectorTokenClient,
         });
-        axiosConfig.headers.Authorization = accessToken;
+
+        if (accessToken) {
```

Comment:
__note:__ handling of the `null` access tokens will be implemented separately in the scope of https://github.com/elastic/kibana/issues/131231 (cc @ymao1)